DisplayCAL/argyll_cgats.py

```diff
@@ -519,8 +523,7 @@ def extract_device_gray_primaries(
             RGB_XYZ_extracted[RGB] = XYZ
         elif RGB not in [(100.0, 100.0, 100.0), (0.0, 0.0, 0.0)]:
             RGB_XYZ_remaining[RGB] = XYZ
-    for RGB in dupes:
-        count = dupes[RGB]
+    for RGB, count in dupes.items():
```

Comment:
using ``items()`` here is okay, but beware that, after the removal of the ``OrderedDict`` the ``items()`` method of the ``dict()`` was not working as expected by the DisplayCAL.